### PR TITLE
Make extensions use typed methods Minz_Request::paramString and Minz_Session::paramString

### PR DESCRIPTION
**Pull request by mkorthof**
As per short discussion in PR [#5830](https://github.com/FreshRSS/FreshRSS/pull/5830):

https://github.com/FreshRSS/FreshRSS/pull/5830#issuecomment-1816813504
https://github.com/FreshRSS/FreshRSS/pull/5830#issuecomment-1817137233

Tested the 3 extensions with actual changes in docker container running `d2aef84827ee freshrss/freshrss:edge`:

- xExtension-CustomJS
- xExtension-CustomCSS
- xExtension-ImageProxy

### CODE REVIEW

File: xExtension-ImageProxy/extension.php [hunk 1]
```diff
@@ -52,12 +54,12 @@ public function handleConfigureAction() {
 		$this->registerTranslates();
 
 		if (Minz_Request::isPost()) {
-			FreshRSS_Context::$user_conf->image_proxy_url = Minz_Request::param('image_proxy_url', self::PROXY_URL, true);
-			FreshRSS_Context::$user_conf->image_proxy_scheme_http = Minz_Request::param('image_proxy_scheme_http', '');
-			FreshRSS_Context::$user_conf->image_proxy_scheme_https = Minz_Request::param('image_proxy_scheme_https', '');
-			FreshRSS_Context::$user_conf->image_proxy_scheme_default = Minz_Request::param('image_proxy_scheme_default', self::SCHEME_DEFAULT);
-			FreshRSS_Context::$user_conf->image_proxy_scheme_include = Minz_Request::param('image_proxy_scheme_include', '');
-			FreshRSS_Context::$user_conf->image_proxy_url_encode = Minz_Request::param('image_proxy_url_encode', '');
+			FreshRSS_Context::$user_conf->image_proxy_url = Minz_Request::paramString('image_proxy_url', true) ?: self::PROXY_URL;
```
Comment (Frenzie) [inline xExtension-ImageProxy/extension.php:57]: Is that null coalescing parameter thing equivalent to isset()?

**Review by Frenzie — COMMENTED** [review]
Fine by me in principle, though I'm a little worried about the addition of strict_types changing behavior? I'd rather have seen that in a separate PR since the param stuff is insta-mergeable. ;-)

File: xExtension-ImageProxy/extension.php (hunk 1 above)
Comment (Alkarex) [reply xExtension-ImageProxy/extension.php:57]: Equivalent to `empty()`

**Comment by Alkarex** [comment]
Looks good 👍🏻 
Would you be able to give a quick try to the extensions, to check that are still running without crash?

**Comment by mkorthof** [comment]
> Looks good 👍🏻 Would you be able to give a quick try to the extensions, to check that are still running without crash?

I'm actually using CustomJS myself so that got the most testing :) It's been working fine so far.
Also enabled the other extensions and tested changing settings of the ones that can be configured (CustomJS, CustomCSS, ImageProxy).

No errors in apache or frss logs and no other issues afaict. Using latest docker image (e4e587aeecf5).

**Comment by mkorthof** [comment]
Did quick test with with Dockerfile-Oldest, no issues.

**Comment by Alkarex** [comment]
Thanks for the tests 👍🏻

**Pull request merged**